Model/CategoryManagerInterface.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Model;
+
+use Sonata\ClassificationBundle\Model\CategoryInterface;
+
+/**
+ * @author Joao Albuquerque <albuquerque.joao.filipe@gmail.com>
+ * @author Christian Gripp <mail@core23.de>
+ */
+interface CategoryManagerInterface
+{
+    /**
+     * @param string $context
+     *
+     * @return CategoryInterface
+     */
+    public function getRootCategory($context);
+
+    /**
+     * @param bool|true $loadChildren
```

Comment:
AFAIK, It is valid phpdoc (https://phpdoc.org/docs/latest/guides/types.html, `bool` and `true` are both valid types), and it means the parameter is a boolean with default value `true`